src/goto-instrument/contracts/contracts.cpp

```diff
@@ -853,18 +856,16 @@ bool code_contractst::check_frame_conditions_function(const irep_idt &function)
     function,
     symbol_table);
 
-  // Adds formal parameters to write set
+  // fetch local statics of all subfunctions in the symbol_table
```

Comment:
Much like the commit message: I don't think "subfunctions" is the right terminology.